client_test/virtualdefund_test.go

```diff
@@ -61,14 +61,14 @@ func runVirtualDefundIntegrationTest(t *testing.T, messageDelay time.Duration, o
 	clientI, storeI := setupClient(irene.PrivateKey, chainServiceI, broker, logDestination, messageDelay)
 
 	numOfVirtualChannels := uint(5)
-	paidToBob := uint(1)
-	totalPaidToBob := paidToBob * numOfVirtualChannels
+	//paidToBob := uint(1)
```

Comment:
Clean this up

Reply:
[5ee1910](https://github.com/statechannels/go-nitro/pull/826/commits/5ee1910a49d6d727b7f12a9dc00386cb50a6f360)